doc/setup.rst

```diff
@@ -7,10 +7,117 @@ Setup
 This document will describe how to organize your
 raw files and the settings files for running PYPIT.
 
+If you tend to observe with one instrument configuration
+and with a simple set of calibrations, then setup should
+be straightforward.  If you use multiple configurations
+(e.g. gratings, grating tilts), then one must pay more
+careful attention to the setup.
 
 Raw Files
 =========
 
+The raw files may be located anywhere on the harddrive.
+We recommend you organize them by date of observation.
+They can be gzipped although Python's FITS reader works
+considerably slower on compressed files.
+
+pypit_setup
+===========
+
+PYPIT includes a script that one may execute (*recommended*)
+to initiate the data reduction process.  This script helps organize
+the primary data reduction process that follows.  It also
+generates a default Settings File which can be edited further
+to run PYPIT.
+
+Execution
+---------
+
+The setup script requires two inputs: the root of the data
+files (with the full path) and the instrument name.  Here is an example::
+
+    pypit_setup /Users/xavier/Keck/LRIS/data/2016apr06/Raw/LB lris_blue
```

Comment:
Yes, this is an excellent suggestion.  I'm doing some additional docs now and will include this.